.github/workflows/add-artifacts-to-release.yml

```diff
@@ -124,12 +124,6 @@ jobs:
         run: |
           ls -lah libcorerust-${{ matrix.target }}
           zip -j babylon-node-rust-${{ matrix.zipname }}-release-${{ github.event.release.tag_name }}.zip libcorerust-${{ matrix.target }}/${{ matrix.artifact }}
-      - if: github.event_name == 'release'
-        name: Get release
-        id: get_release
-        uses: bruceadams/get-release@74c3d60f5a28f358ccf241a00c9021ea16f0569f
```

Comment:
This repo is not supported anymore and uses the outdated set-output that was originially gonna be removed starting end of june.